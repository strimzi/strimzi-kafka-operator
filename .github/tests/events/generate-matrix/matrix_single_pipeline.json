{
  "event_name": "workflow_dispatch",
  "action": "workflow_dispatch",
  "inputs": {
    "pipelines": "upgrade",
    "profiles": ""
  },
  "repository": {
    "name": "ci-playground",
    "owner": { "login": "frawless" }
  }
}