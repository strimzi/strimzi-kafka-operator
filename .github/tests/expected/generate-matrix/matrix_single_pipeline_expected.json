[
  {
    "agent": "oracle-vm-4cpu-16gb-x86-64",
    "arch": "amd64",
    "pipeline": "upgrade",
    "profile": "azp_kraft_upgrade",
    "timeout": 180,
    "strimzi_feature_gates": "false",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 1,
    "jobName": "upgrade-azp_kraft_upgrade-amd64"
  },
  {
    "agent": "oracle-vm-4cpu-16gb-x86-64",
    "arch": "amd64",
    "pipeline": "upgrade",
    "profile": "azp_kafka_upgrade",
    "timeout": 180,
    "strimzi_feature_gates": "false",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 1,
    "jobName": "upgrade-azp_kafka_upgrade-amd64"
  },
  {
    "agent": "oracle-vm-4cpu-16gb-arm64",
    "arch": "arm64",
    "pipeline": "upgrade",
    "profile": "azp_kraft_upgrade",
    "timeout": 180,
    "strimzi_feature_gates": "false",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 1,
    "jobName": "upgrade-azp_kraft_upgrade-arm64"
  },
  {
    "agent": "oracle-vm-4cpu-16gb-arm64",
    "arch": "arm64",
    "pipeline": "upgrade",
    "profile": "azp_kafka_upgrade",
    "timeout": 180,
    "strimzi_feature_gates": "false",
    "strimzi_rbac_scope": "CLUSTER",
    "cluster_operator_install_type": "yaml",
    "parallel": 1,
    "jobName": "upgrade-azp_kafka_upgrade-arm64"
  }
]